pilot/pkg/networking/core/v1alpha3/cluster_builder.go

```diff
@@ -592,33 +613,58 @@ func addUint32(left, right uint32) (uint32, bool) {
 func (cb *ClusterBuilder) buildInboundPassthroughClusters() []*cluster.Cluster {
 	// ipv4 and ipv6 feature detection. Envoy cannot ignore a config where the ip version is not supported
 	clusters := make([]*cluster.Cluster, 0, 2)
-	if cb.supportsIPv4 {
-		inboundPassthroughClusterIpv4 := cb.buildDefaultPassthroughCluster()
-		inboundPassthroughClusterIpv4.Name = util.InboundPassthroughClusterIpv4
-		inboundPassthroughClusterIpv4.Filters = nil
-		inboundPassthroughClusterIpv4.UpstreamBindConfig = &core.BindConfig{
+	// if Dual Stack enables
+	if features.EnableDualStack {
+		inboundPassthroughCluster := cb.buildDefaultPassthroughCluster()
+		inboundPassthroughCluster.Name = util.InboundPassthroughClusterIpv4
```

Comment:
The name hear seems misleading/wrong

Reply:
okay, I changed `inboundPassthroughCluster` as `inboundPassthroughClusterForDS`

Reply:
My issue was not the internal go variable name, but the external Envoy name InboundPassthroughClusterIpv4

Reply:
okay, so I will change `inboundPassthroughCluster.Name` from `InboundPassthroughClusterIpv4` to `inboundPassthroughClusterForDualStack`.